src/components/NavTitleContainer.vue

```diff
@@ -36,6 +36,7 @@ export default {
 }
 
 .title {
+  color: var(--color-nav-root-title, currentColor);
```

Comment:
I think if you dont specify a default, and there is no css var for it, it would just work as if the rule was never defined, right?

Reply:
I think so, but this one has a default (`currentColor`). Or am I misunderstanding what you're asking maybe?